doc/nrf/ug_matter_hw_requirements.rst

```diff
@@ -162,3 +166,348 @@ Values are provided in kilobytes (KB).
       +------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+-------------+
 
 ..
+
+.. _ug_matter_hw_requirements_layouts:
+
+Reference Matter memory layouts
+*******************************
+
+The following sections show how the conceptual :ref:`ug_matter_overview_architecture` translates to actual memory maps for each of the available :ref:`ug_matter_overview_architecture_integration_designs`.
```

Comment:
It's a nit, but `ug_matter_overview_architecture` provides just an abstract view of the Matter stack, so it doesn't really affect the memory map anyhow. Shouldn't we rather link to `ug_matter_overview_architecture_integration` or whatever describes Matter in NCS?